testy/studentCode.py

```diff
@@ -8,9 +8,20 @@ def setup(pipe):
   # state[0] = startupValue
 
 def main(stateQueue, pipe):
-  while True:
-    pass
+  # while True:
+  #   pass
+  stateQueue.put([SM_COMMANDS.GET_VAL, "incrementer"])
+  response = pipe.recv()
+  print("Get Info:", response)
+  response -= 1
+
+  stateQueue.put([SM_COMMANDS.SET_VAL, "incrementer", response])
+  response = pipe.recv()
+  print("Set info:", response)
+
+  # stateQueue.put([SM_COMMANDS.GET_VAL, "float1"])
+  # response = pipe.recv()
+  # print("Get Info:", response)
+
   print("Saying hello to the other side")
-  state = Robot.getValue("")
-  print("Hello from the other side %d" % (state,))
-  1.0/state
+  print("DAT:", 1.0/response)
```

Comment:
Add newline at end of file. 
Has semantic meaning apparently; google for more details.